python/ql/lib/semmle/python/Concepts.qll

```diff
@@ -443,6 +443,41 @@ module RegexExecution {
   }
 }
 
+/**
```

Comment:
I'm thinking we should create an `Ldap` module that will contain all LDAP related concepts, like we've done for HTTP related concepts. I know that we have multiple experimental queries, so I think it could be worth the effort :+1:

Reply:
I am happy to do that, it was like that in the external contribution, actually  :-)

Reply:
Oh, I misunderstood. Yes, that is a good idea.

Reply:
Done